src/SmartComponents/Inventory/applications/vulnerabilities/VulnerabilitiesCveTableToolbar.js

```diff
@@ -13,9 +13,9 @@ import { addNotification } from '../../../Notifications';
 
 const CVSSOptions = [
     { value: 'all', label: 'All', disabled: false, from: '', to: '' },
-    { value: '0to3', label: 'Lower than 3', disabled: false, from: '', to: 3 },
-    { value: '3to7', label: '3 - 7', disabled: false, from: 3, to: 7 },
-    { value: '7to10', label: '7 - 10', disabled: false, from: 7, to: 10 }
+    { value: '0to3', label: 'CVSS < 3.0', disabled: false, from: '', to: 3 },
```

Comment:
3 is both in the first and second interval. It should be in the second one only.